test/extended/include.go

```diff
@@ -12,18 +12,12 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/instrumentation"
 	_ "k8s.io/kubernetes/test/e2e/kubectl"
 
-	// _ "k8s.io/kubernetes/test/e2e/lifecycle"
```

Comment:
Sync with @smarterclayton before you remove it from here, I'm not sure if these need to brought back or not, I see that he added them commented out from the start, so better to double check.

Reply:
I think that these tests were disabled 2 years ago :) https://github.com/openshift/origin/commit/76d0cb1bbaf44761638f4886614ee8a01c2ae749